src/main/resources/generator/server/springboot/mvc/security/oauth2/core/main/infrastructure/primary/CustomClaimConverter.java.mustache

```
@@ -82,14 +83,18 @@ class CustomClaimConverter implements Converter<Map<String, Object>, Map<String,
       HttpHeaders headers = new HttpHeaders();
       headers.set(HttpHeaders.AUTHORIZATION, buildBearer(getToken(attributes)));
 
-      ResponseEntity<ObjectNode> userInfo = restTemplate.exchange(
-        registration.getProviderDetails().getUserInfoEndpoint().getUri(),
-        HttpMethod.GET,
-        new HttpEntity<String>(headers),
-        ObjectNode.class
-      );
-
-      return userInfo.getBody();
+      try {
+        ResponseEntity<ObjectNode> userInfo = restTemplate.exchange(
+          registration.getProviderDetails().getUserInfoEndpoint().getUri(),
+          HttpMethod.GET,
+          new HttpEntity<String>(headers),
+          ObjectNode.class
+        );
+
+        return userInfo.getBody();
+      } catch (ResourceAccessException ex) {
```

It's better to rethrow the exception in this case I think

---

I agree! I pushed a new version. Thank you @DamnClin